eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionReportWriterTest.java

```diff
@@ -164,160 +172,140 @@ public static String getFirstVariantLine(File output) throws IOException {
-                getFirstVariantLine(output));
-    }
-
-    private void writeIndelWithAccessionHelper(String reference, String alternate, String denormalizedReference,
-                                               String denormalizedAlternate) throws IOException {
-        SubmittedVariant variant = new SubmittedVariant("accession", TAXONOMY, "project", CONTIG_1, START_1, reference,
-                                                        alternate, CLUSTERED_VARIANT, SUPPORTED_BY_EVIDENCE,
-                                                        MATCHES_ASSEMBLY, ALLELES_MATCH, VALIDATED, null);
-
-        AccessionWrapper<ISubmittedVariant, String, Long> accessionWrapper = new AccessionWrapper<>(ACCESSION, "1",
-                                                                                                    variant);
-
-        AccessionWrapperComparator accessionWrapperComparator = new AccessionWrapperComparator(
-                Collections.singletonList(variant));
-
-        accessionReportWriter.write(Collections.singletonList(accessionWrapper), accessionWrapperComparator);
-
-        assertEquals(String.join("\t", CHROMOSOME_1, Integer.toString(START_1 - 1), ACCESSION_PREFIX + ACCESSION,
-                                 denormalizedReference, denormalizedAlternate,
-                                 ".", ".", "."),
-                     getFirstVariantLine(output));
+        writeVariantWithAccessionHelper(1, "", "A", 1, "G", "AG");
     }
 
     @Test
     public void writeDeletionWithAccession() throws IOException {
-        writeIndelWithAccessionHelper(REFERENCE, "", CONTEXT_BASE + REFERENCE, CONTEXT_BASE);
+        writeVariantWithAccessionHelper(START_1, REFERENCE, "", START_1 - 1, CONTEXT_BASE + REFERENCE, CONTEXT_BASE);
     }
 
     @Test
     public void writeDeletionWithAccessionFirstPositionStart() throws IOException {
-        SubmittedVariant variant = new SubmittedVariant("accession", TAXONOMY, "project", CONTIG_1, 1, "G",
-                                                        "", CLUSTERED_VARIANT, SUPPORTED_BY_EVIDENCE,
-                                                        MATCHES_ASSEMBLY, ALLELES_MATCH, VALIDATED, null);
-
-        AccessionWrapper<ISubmittedVariant, String, Long> accessionWrapper =
-                new AccessionWrapper<ISubmittedVariant, String, Long>(ACCESSION, "1", variant);
-
-        AccessionWrapperComparator accessionWrapperComparator = new AccessionWrapperComparator(
-                Collections.singletonList(variant));
-
-        accessionReportWriter.write(Collections.singletonList(accessionWrapper), accessionWrapperComparator);
-
-        assertEquals(
-                String.join("\t", CHROMOSOME_1, "1", ACCESSION_PREFIX + ACCESSION, "GT", "T", ".", ".", "."),
-                getFirstVariantLine(output));
+        writeVariantWithAccessionHelper(1, "G", "", 1, "GT", "T");
```

Comment:
Why not use the symbolic variables as in the previous method?

Reply:
oversight, and wasn't sure the variables added value. will use them anyway for consistency